Jasmine/Common/ViewControllers/GameElements/SquareGridViewController.swift

```diff
@@ -15,10 +15,14 @@ class SquareGridViewController: UIViewController {
 
     // MARK: Properties
     /// Stores the maximum number of rows that should be displayed in this view.
-    fileprivate var numRows = 0
+    fileprivate var numRows: Int {
+        return collectionData.numRows
```

Comment:
No. UICollectionView might throw an error because number of rows, columns cannot be dynamically changed.